src/managed/CommonManaged.props

```diff
@@ -17,6 +19,26 @@
     <PackageThirdPartyNoticesFile>$(RepoRoot)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
   </PropertyGroup>
 
+  <!--
+  Once https://github.com/dotnet/buildtools/issues/1548 is fixed, the following version sections should be simply
+  <Import Condition="Exists('$(ToolsDir)BuildVersion.targets')" Project="$(ToolsDir)BuildVersion.targets" />
+  -->
+  <PropertyGroup>
+    <TodayTimeStamp>$([System.DateTime]::Now.ToString(yyyyMMdd))</TodayTimeStamp>
+    <BuildVersionFile Condition="'$(BuildVersionFile)'==''">$(ObjDir)BuildVersion-$(TodayTimeStamp).props</BuildVersionFile>
+  </PropertyGroup>
+
+  <Import Condition="Exists('$(BuildVersionFile)')" Project="$(BuildVersionFile)" />
```

Comment:
There should be a shim file in BuildTools that you import which determines the `BuildVersionFile` property and then imports the right file.  That would allow repo's to just import the single `BuildVersionFile` and not carry around this logic to determine the build version file name.  @eerhardt , I think we talked about this a while ago.  It's definitely unrelated to what you've merged here though and isn't blocking.

Thanks for getting this committed.